include/nicegraf.h

```diff
@@ -3306,6 +3306,114 @@ void ngf_cmd_write_image(
  */
 ngf_error ngf_cmd_generate_mipmaps(ngf_xfer_encoder xfenc, ngf_image img) NGF_NOEXCEPT;
 
+/**
+ * \ingroup ngf
+ * The RenderDoc API version to be used
+ */
+#define NGF_RENDERDOC_API_VERSION 1_6_0
+
+/**
+ * \ingroup ngf
+ * Helper to obtain a RenderDoc API interface with \ref NGF_RENDERDOC_API_VERSION. 
+ */
+#define NGF_RENDERDOC_API NGFI_EVAL_AND_PASTE(RENDERDOC_API_, NGF_RENDERDOC_API_VERSION)
+
+/**
+ * \ingroup ngf
+ * Helper to obtain a RenderDoc API enum with \ref NGF_RENDERDOC_API_VERSION. 
+ */
+#define NGF_RENDERDOC_VERSION NGFI_EVAL_AND_PASTE(eRENDERDOC_API_Version_, NGF_RENDERDOC_API_VERSION)
+
```

Comment:
These defines don't need to go into the public header. The end user shouldn't care what renderdoc version of the renderdoc api we're on.